cli/tools/test.rs

```diff
@@ -452,7 +452,7 @@ impl TestReporter for PrettyTestReporter {
   fn report_summary(&mut self, summary: &TestSummary, elapsed: &Duration) {
     if !summary.failures.is_empty() {
       let mut failure_titles = vec![];
-      println!("\nfailures:\n");
+      println!("\n{}\n", colors::white_bold_on_red("  ERRORS  "));
```

Comment:
Is it intentional that `ERRORS` has double spaces around it while `FAILURES`  has single spaces? I would prefer single like jest for both...

Reply:
Yeah, I wanted them to be the same width

Reply:
I believe this style is supposed to be more like red-highlighted text rather than uniform red text boxes. The spaces are to add fixed horizontal padding since ansi backgrounds happen to look ugly without it. At least, it looks nicer that way IMO

Reply:
Can you explain a bit more what you are proposing?

Reply:
```suggestion
      println!("\n{}\n", colors::white_bold_on_red(" ERRORS "));
```
And to shift the mental model to 'red-highlighted text with single-space padding' rather than worrying about the red boxes being the same width.